models/src/anemoi/models/commands/migration.py

```diff
@@ -0,0 +1,431 @@
+            if len(executed_migrations):
+                print(
+                    len(executed_migrations),
+                    " registered ",
+                    maybe_plural(len(executed_migrations), "migration"),
+                    ":",
+                    sep="",
+                )
+                console.print("  [italic]These migrations are already executed and part of the checkpoint[/italic]")
+            for migration in executed_migrations:
+                console.print(
+                    f"  [cyan]* [bold]{migration.name}[/bold] \\[v{migration.metadata.versions['anemoi-models']}][/cyan]"
+                )
+            if len(extra_migrations):
+                print(
+                    len(extra_migrations),
+                    "extra",
+                    maybe_plural(len(extra_migrations), "migration"),
+                    "to rollback:",
+                )
+            for migration in extra_migrations:
+                console.print(
+                    f"  [red]+ [bold]{migration.name}[/bold] \\[v{migration.metadata.versions['anemoi-models']}][/red]"
+                )
+            if len(missing_migrations):
+                print(
+                    len(missing_migrations),
+                    " missing ",
+                    maybe_plural(len(missing_migrations), "migration"),
+                    ":",
+                    sep="",
+                )
+            for migration in missing_migrations:
+                console.print(
+                    f"  [green]+ [bold]{migration.name}[/bold] \\[v{migration.metadata.versions['anemoi-models']}][/green]"
+                )
+            if len(missing_migrations) or len(extra_migrations):
+                console.print("\n[italic]To update your checkpoint, run:[/italic]")
+                console.print(f"  [italic]anemoi-models migration sync {args.ckpt}[/italic]")
+        except IncompatibleCheckpointException as e:
+            print(str(e))
+
+    def run_fix_order(self) -> None:
+        """Fixes the order of the new migration scripts.
+        It uses the earliest possible time with the last migration name in origin/main.
+        """
+        new_migrations = new_migrations_from_main_branch()
+        all_migrations = sorted(
+            [file.name for file in MIGRATION_PATH.iterdir() if file.is_file() and file.name != "__init__.py"]
+        )
+        incorrect_order, last_upstream_name = in_incorrect_order(all_migrations, new_migrations)
+
+        if last_upstream_name is None or not len(incorrect_order):
+            print("No migration to rename.")
+            return
+
+        new_timestamp = int(last_upstream_name.partition("_")[0]) + 1
+        for k, name in enumerate(new_migrations):
+            path = MIGRATION_PATH / name
+            _, _, new_name = name.partition("_")
+            new_name = f"{new_timestamp + k}_{new_name}"
+            print(f"Renaming {name} to {new_name}.")
```

Comment:
I noticed there is some mixing of print and LOGGER statements. I'd propose to replace the print statements and just use the LOGGER?